src/components/content/Content.js

```diff
@@ -1,40 +1,36 @@
 import React from 'react';
 
 import Hello from '../hello';
-import Counter from '../counter';
-import Numbers from '../numbers';
-import Button from '../button';
-import Show from '../show';
 import Date from '../clock';
-import Scintillation from '../scintillation';
+import Products from '../../pages/products';
+import Login from '../../pages/login';
 
 import './content.scss';
+import Home from "../../../../bufer21/src/components/main/Main";
 
 
 const Error = (props) =>
```

Comment:
Avoid to use names of native constructors.
If you don't need this component here, remove it. Or move to separate module.